Marlin/src/module/stepper.cpp

```diff
@@ -2241,13 +2248,13 @@ hal_timer_t Stepper::calc_timer_interval(uint32_t step_rate) {
   #endif // !CPU_32_BIT
 }
 
-#if ENABLED(NONLINEAR_EXTRUSION)
+#if ENABLED(NONLINEAR_EXTRUSION) and DISABLED(SMOOTH_LIN_ADVANCE)
```

Comment:
What is this? Please review code for left behind "concepts" before and after submission. Did AI generate this code??